projects/fizzbuzz/java/README.md

```diff
@@ -1 +1,8 @@
-Place java solutions to fizz buzz in this directory
+Project 2: Fuzz Buzz
+
+Description: An integer is input and the number 1 from n is printed out. 
+However, if the number is a multiple of 3 the word "fizz" is printed instead of the number.
+If the number is a multiple of 5 the word "buzz" is printd instead of the number.
+
+Instructions: When prompted by the program enter an integer.  
+
```

Comment:
+1
The code is very good, the instructions explain the problem well. Could be good to give instructions on set up and running for expanded user base.